Docker/itk-base/Dockerfile

```diff
@@ -16,7 +17,11 @@ RUN curl -O https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz && \
     cd /tmp/cmake-build && \
     ../cmake-3.4.3/bootstrap && \
     make -j$(nproc) && \
-    ./bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release . && \
+    cd .. && \
```

Comment:
Indeed it works if you download `pip` with `get-pip.py` but not if you install the system version (which cannot be updated with it seems)